cocos/ui/UIScale9Sprite.cpp

```diff
@@ -794,7 +794,7 @@ namespace ui {
         //shrink the image size when it is 9-patch
         if(_isPatch9)
         {
-            float offset = 1.4f;
```

Comment:
@pandamicro  Android 9-patch image has extra 1 pixel should be removed.
The previous 1.4 is a magic number.